apps/cli-daemon/src/app/workspace/workspace.service.ts

```diff
@@ -15,10 +15,14 @@ import {
   Logger,
   NotFoundException,
 } from '@nestjs/common';
+import { JSONSchema7 } from 'json-schema';
+import { Draft07 } from 'json-schema-library';
+import * as angularSchema from 'node_modules/@angular/cli/lib/config/schema.json';
```

Comment:
why this import is from 'node_modules' and not from @angular/cli like the others?